.travis.yml

```diff
@@ -8,6 +8,26 @@ rvm:
   - jruby-9.1.7.0
   - jruby-head
 matrix:
```

Comment:
With the matrix you don't need the `rvm:` section above at all. The matrix explicitly says what to build.